snuba/migrations/snuba_migrations/functions/0001_functions.py

```diff
@@ -0,0 +1,189 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import (
+    UUID,
+    AggregateFunction,
+    Array,
+    Column,
+    DateTime,
+    Float,
+    Nested,
+    String,
+    UInt,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+common_columns: List[Column[Modifiers]] = [
+    Column("org_id", UInt(64)),
+    Column("project_id", UInt(64)),
+    Column("transaction_name", String()),
+    Column("timestamp", DateTime()),
+    Column("depth", UInt(32)),
+    Column("parent_fingerprint", UInt(64)),
+    Column("fingerprint", UInt(64)),
+    Column("symbol", String()),
+    Column("image", String()),
+    Column("filename", String()),
+    Column("is_application", UInt(8)),
```

Comment:
How do you plan to use these additional fields.

Reply:
Can you elaborate a little more on how this `materialization_version` is used? If I understand it correctly, if we want to populate a new materialization with new columns, we would have to do a migration to insert the column on the raw table as well. Is the use here just so that we can target only rows with the right `materialization_version` when the table is materialized?

Reply:
When you want to move to a new materialization view having a `materialization_version` would allow you to gracefully do it. Lets assume you have a `fn_raw_table` with `materialization_version` fields being written with value 1. It populates `fn_mv_1` based on `WHERE materialization_version = 1`. Now you have a use case where you add another column to `fn_raw_table`, say column `x`. You want to build a new materialization using column `x`. So you define `fn_mv_2` using `WHERE materialization_version=2`. Doing this allows you to alter the raw table and populate new materializations.